lms/resources/_js_config.py

```diff
@@ -0,0 +1,100 @@
+import datetime
+import urllib
+
+import jwt
+
+from lms.validation.authentication import BearerTokenSchema
+
+
+class JSConfig:
```

Comment:
Is this `JSConfig`, or is it `LTILaunchJSConfig` etc.?

Reply:
I think just `JSConfig` for now as it's the one and only JavaScript config object (which happens only to be used in LTI launches currently). If we ever need JS config for non-LTI launch requests, I don't know yet whether that would be handled by this class or a completely different class